discovery-provider/src/api/v1/tracks.py

```diff
@@ -38,9 +47,27 @@ def get(self, track_id):
 
 @ns.route("/<string:track_id>/stream")
 class TrackStream(Resource):
+    @ns.doc(
+        id="""Stream Track""",
+        params={'track_id': 'A Track ID'},
+        responses={
+            200: 'Success',
+            216: 'Partial content',
+            400: 'Bad request',
+            416: 'Content range invalid',
+            500: 'Server error'
+        }
+    )
     @cache(ttl_sec=5)
     def get(self, track_id):
-        """Redirect to track mp3"""
+        """
+        Get the track's streammable mp3 file.
```

Comment:
typo: `streammable`
this is your favorite one!!

Reply:
gosh